lib/fastlane/plugin/wpmreleasetoolkit/models/firebase_test_lab_result.rb

```diff
@@ -0,0 +1,36 @@
+module Fastlane
+  class FirebaseTestLabResult
+    def initialize(log_file_path:)
+      raise "No log file found at path #{log_file_path}" unless File.file? log_file_path
+
+      @path = log_file_path
+    end
+
+    # Scan the log file to for indications that the Test Run failed
+    def success?
+      File.readlines(@path).any? { |line| line.include? '0 test cases failed' }
```

Comment:
I wonder if there's a risk that a given FTL run could run multiple test suites, and the logs could then contain stuff like `Test Suite 1: 2 test cases failed` while later in the logs it would also contain `Test Suite 2: 0 test cases failed`… making this whole thing a false positive? 🤔

Reply:
We only run tests one suite at at time right now – I'm fine to adjust this down the road if it needs more correctness tbh 😅